js/app.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+(function(){
+  angular
+  .module("wdi_radio", [
+    "ui.router",
+    "songs"
+    ])
+  .config([
+    "$stateProvider",
+    RouterFunction
+  ]);
+
+  function RouterFunction($stateProvider){
+    $stateProvider
+    .state("songIndex", {
+      url: "/songs",
+      templateUrl: "js/songs/index.html",
+      controller: "SongIndexController",
+      controllerAs: "SongIndexVM"
+    })
+    .state("songNew", {
```

Comment:
Great job with your `RouterFunction` and defining different states, everything looks great :+1: